projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/graphql/strategy/datajob/JobFieldStrategyByTeam.java

```diff
@@ -41,7 +42,8 @@ public Criteria<V2DataJob> computeFilterCriteria(
                 part = part.replace("%", ".*").trim();
                 var configTeamLower = config.getTeam().toLowerCase();
                 var partLower = part.toLowerCase();
-                return checkMatch(configTeamLower, partLower);
```

Comment:
All code changes are reverting to previous code before wildcard ('*) matching was introduced in the PR mentioned in the description. The old implementation uses regex for matching and could have different behavior in certain edge cases, seeing as there was some confusion among users of the api I thought it best to just revert the code back to what it used to be, except for the computeFilterCriteria in the JobFieldStrategyByName class. 

As for using the FilenameUtils method, It can be a bit misleading, as it may not immediately be clear why a file-related method is being used for this purpose.
The use of FilenameUtils.wildcardMatch for matching team names and job names, was used because it is a convenient and efficient way to perform wildcard matching, regardless of the type of string being matched. It was already available, rather than writing a custom function from scratch.